hack/build/push_image.sh

```diff
@@ -7,24 +7,16 @@ if [[ ! "${TAG}" ]]; then
 fi
 
 commit=$(git rev-parse HEAD)
```

Comment:
does `GITHUB_SHA` (default env variable) result in the same value?

Reply:
this script does not run in our github ci, therefore `GITHUB_SHA` does not exist here